pkg/apiserver/compression.go

```diff
@@ -54,58 +55,78 @@ func configureCompression(router *gin.Engine, settings CompressionSettings) erro
 	}
 
 	router.Use(ginGzip.Gzip(level, opts...))
+	router.Use(recordResponseSize)
```

Comment:
does it make sense to set this here? I would think that you want to record the request/response size no matter whether compression is anebled or not.

Reply:
We are already recording the response size by default in the field `bytes`, so we only need to record the uncompressed size when we are compressing (and `bytes` will then carry the compressed size)